stix_shifter_modules/rhacs/stix_translation/query_constructor.py

```diff
@@ -0,0 +1,506 @@
+""" This Module will convert Stix Pattern to RHACS data source supported query """
+import re
+import json
+import logging
+from os import path
+from datetime import datetime, timedelta
+from stix_shifter_utils.stix_translation.src.patterns.pattern_objects import ObservationExpression, \
+    ComparisonExpression, ComparisonComparators, Pattern, \
+    CombinedComparisonExpression, CombinedObservationExpression
+
+logger = logging.getLogger(__name__)
+
+START_STOP_PATTERN = r"(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(\.\d+)?Z)"
+TYPE_MAP_PATH = "json/type_map.json"
+
+STOP_TIME = datetime.utcnow()
+
+
+class StartStopQualifierValueException(Exception):
+    """ Start Stop qualifier exception """
+    pass
+
+
+class FileNotFoundException(Exception):
+    """ file not found exception """
+    pass
+
+
+class QueryStringPatternTranslator:
+    """
+    translate stix pattern to native data source query language
+    """
+
+    def __init__(self, pattern: Pattern, data_model_mapper, options):
+        logger.info("RHACS Connector")
+        self.dmm = data_model_mapper
+        self.options = options
+        self.timeframe = []
+        self.comparator_lookup = self.dmm.map_comparator()
+        self.type_map = self.load_json(TYPE_MAP_PATH)
```

Comment:
use absolute path to read files from a different folder.

Reply:
Updated load_json function to use absolute path as suggested.